### PR TITLE
LessonCard-LessonsPage-TopicPages-API-call-and-CSS-fix

### PR DESCRIPTION
**Pull request by Dash293**

### CODE REVIEW

**Review by ThomasDCW — APPROVED** [review]

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
No review SM

**Review by ThomasDCW — APPROVED** [review]

File: frontend/src/components/AudioPlayerLoading/index.jsx [hunk 1]
```diff
@@ -18,7 +18,7 @@ export default function AudioPlayerLoading() {
   const [playOrPauseImg, setPlayOrPauseImg] = useState(playImg);
 
   useEffect(() => {
-    axios.get("http://localhost:5000/lessons/:id").then(({ data }) => {
+    axios.get("http://localhost:5000/lessons/").then(({ data }) => {
```
Comment (LuigiMenez) [inline frontend/src/components/AudioPlayerLoading/index.jsx:21]: Use .env variable

File: frontend/src/components/LessonCard/index.jsx [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import axios from "axios";
+import React, { useState, useEffect } from "react";
+import SLessonCard from "./style";
+import Play from "../../assets/play-solid.svg";
+import EmptyHeart from "../../assets/Images/emptyheart.svg";
+
+export default function LessonCard() {
+  const [lessons, setLessons] = useState([]);
+  useEffect(() => {
+    axios.get("http://localhost:5000/lessons").then(({ data }) => {
```
Comment (LuigiMenez) [inline frontend/src/components/LessonCard/index.jsx:10]: same

File: frontend/src/components/LessonCard/index.jsx [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <SLessonCard>
+      <>
+        {lessons.map((lesson) => (
+          <div key={lesson.id} className="CardOfLesson">
+            <img className="play" src={Play} alt="" />
+            <div className="informations">
+              <h1 className="lesson"> {lesson.title}</h1>
+              <h2>Hip-Hop/{lesson.duration}</h2>
+            </div>
+            <img className="heart" src={EmptyHeart} alt="" />
```
Comment (LuigiMenez) [inline frontend/src/components/LessonCard/index.jsx:24]: alt for accessibility

File: frontend/src/components/LessonCard/style.jsx [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  .CardOfLesson {
+    display: flex;
+    align-items: center;
+  }
+  .informations {
+    margin: 10px;
+    width: 70%;
+  }
+  .play {
+    /* border: solid 1px red; */
```
Comment (LuigiMenez) [inline frontend/src/components/LessonCard/style.jsx:13]: no dead code

File: frontend/src/components/LessonCard/style.jsx [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+    overflow: hidden;
+  }
+  h2 {
+    font: var(--fontSecondary);
+    align-items: center;
+    font-size: 1.1rem;
+    font-weight: 300;
+  }
+  .heart {
+    /* border: solid 1px red; */
```
Comment (LuigiMenez) [inline frontend/src/components/LessonCard/style.jsx:34]: same

File: frontend/src/pages/Home/index.jsx [hunk 6]
```diff
@@ -71,5 +71,3 @@ function Home() {
     </SHome>
   );
 }
```
Comment (LuigiMenez) [inline frontend/src/pages/Home/index.jsx:73]: The components is not export??

**Review by LuigiMenez — CHANGES_REQUESTED** [review]
Attention to proofreading

**Review by ThomasDCW — APPROVED** [review]

**Review by ThomasDCW — APPROVED** [review]

**Pull request closed without merging**